src/domain/dataset-update-flow/src/entities/update/update_trigger.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric::{AccountID, AccountName, DatasetID};
+
+use crate::*;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum UpdateTrigger {
```

Comment:
# General: Enum naming
I suggest using this naming scheme:
```rust
enum Enum {
    VariantA(EnumVariantA),
    VariantB(EnumVariantB),
}
```

This removes a lot of creativity from naming things and great for autocomplete.

Suggested renames:

`UpdateManualTrigger` -> `UpdateTriggerManual`
`UpdateAutoPollingTrigger` -> `UpdateTriggerAutoPolling`
`UpdatePushTrigger` -> `UpdateTriggerPush`
`TimeDeltaSchedule` -> `ScheduleTimeDelta`
`CronExpressionSchedule` -> `ScheduleCronExpression`
etc.

Reply:
Accepted as a rule

Reply:
Note: currently events in Task aggregate violate this rule.